### ISSUE TITLE
Gamma initialization?

### ISSUE THREAD

**Issue by cooijmanstim** [iassael/torch-bnlstm#5]
Hey Yannis, thanks for implementing this in Torch. I wonder though if you maybe forgot the initialization of gamma to 0.1? Torch's BatchNormalization doesn't seem to allow specifying this initialization, but equivalently you could just multiply its output by 0.1.

Also we found that the moving average for the population statistics may require tweaking. For our results we didn't want to be confounded by this so we always simply computed the population statistics in an extra pass over the training set. Does your plot show performance using batch statistics or population statistics?

**Comment by iassael** [comment]
Hi @cooijmanstim, I don't think it's exactly, the same, but in my experiments I initialised the weights with bn_layer.weights:fill(0.1). How does that look to you?

**Comment by cooijmanstim** [comment]
I don't know Torch, but that looks like it will do the same thing. :-) How about the plot, did you use batch statistics or population statistics?

**Comment by iassael** [comment]
Each step had its own statistics as reported in the paper.
